members/views/volunteerEmail.py

```diff
@@ -0,0 +1,37 @@
+from django.urls import reverse
+from django.http import HttpResponseRedirect
+from django.shortcuts import render
+from django.views.decorators.clickjacking import xframe_options_exempt
+
+from members.forms import vol_emailForm
+from members.models.department import Department
+
+
+@xframe_options_exempt
+def volunteerEmail(request):
+    if request.method == "POST":
+        # figure out which form was filled out.
+        if request.POST["form_id"] == "vol_email":
+            # email form has been filled
+            vol_email = vol_emailForm(request.POST)
+            if vol_email.is_valid():
```

Comment:
Igen en lidt ligegyldig ting, men du kan sætte if sætnings sammen så der ikke er tre efter hinanden. Python har short circuit på `and` så den kaster ikke undtagelser hvis det første fejler. Så enten: 
```python
if request.method == "POST" and request.POST["form_id"] == "vol_email":
    vol_email = vol_emailForm(request.POST)
    if vol_email.is_valid():
        ...
```
Efter python 3.8 kan vi også gøre sådan det med "Walrus operator" 
https://docs.python.org/3/whatsnew/3.8.html

```python
if (
 request.method == "POST" 
 and request.POST["form_id"] == "vol_email":
 and (vol_email := vol_emailForm(request.POST)).is_valid()
):
    ...
```
Men det er nok en smagssag hvad man kan lide.